TypeCobol.LanguageServer/TypeCobolServerHost.cs

```diff
@@ -295,7 +309,9 @@ static int Main(string[] args)
                 typeCobolServer.UseSyntaxColoring = UseSyntaxColoring;
                 typeCobolServer.UseOutlineRefresh = UseOutlineRefresh;
                 typeCobolServer.UseCfgDfaDataRefresh = UseCfg;
-
+#if EUROINFO_RULES
+                typeCobolServer.CpyCopyNamesMapFilePath = CpyCopyNamesMapFilePath ?? TypeCobolConfiguration.DefaultCopyNameFile;
```

Comment:
I think the use of the default file should be handled directly in `TypeCobolOptions`. It would be factorized for both CLI and LanguageServer.

Reply:
We cannot decide that TypeCobolOptions should set itself to default.

Reply:
Here is how I see the logic:
- if the user don't use the option, use the default path
- if the path (relative or absolute) is invalid, log a message and continue with an empty CPY table
- if the path is correct, load the file

All of this can be implemented in `TypeCobolOptions`. I think the logic should be the same for CLI and LanguageServer.